semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/languagespecific/fuzzyc/MethodStubCreator.scala

```diff
@@ -13,44 +13,26 @@ case class NameAndSignature(name: String, signature: String)
 /**
   * This pass has no other pass as prerequisite.
   */
-class MethodStubCreator(cpg: Cpg) extends ParallelCpgPass[(NameAndSignature, Int)](cpg) {
+class MethodStubCreator(cpg: Cpg) extends CpgPass(cpg) {
 
   // Since the method fullNames for fuzzyc are not unique, we do not have
   // a 1to1 relation and may overwrite some values. We deem this ok for now.
-  private var methodFullNameToNode = Map[String, nodes.MethodBase]()
-  private var methodToParameterCount = Map[NameAndSignature, Int]()
-
-  override def init(): Unit = {
-    cpg.method
-      .sideEffect { method =>
-        methodFullNameToNode += method.fullName -> method
-      }
-      .exec()
-
-    cpg.call
-      .sideEffect { call =>
-        methodToParameterCount +=
-          NameAndSignature(call.name, call.signature) -> call._astOut.asScala.size
+  private val methodFullNameToNode = scala.collection.mutable.HashMap[String, nodes.MethodBase]()
```

Comment:
Please stop using fully qualified class names. There is a reason why the `import` statement exists...
So please use `mutable.HashMap` like most of the other code.